index.js

```diff
@@ -44,14 +44,16 @@ module.exports = {
           fileBase: context.distDir,
           filePaths: filesToUpload,
           gzippedFilePaths: gzippedFiles,
-          metadata: metadata
+          metadata: metadata,
+          gcloud: {},
         };
 
-        if (projectId && credentials) {
-          config['gcloud'] = {
-            credentials: credentials,
-            projectId: projectId
-          };
+        if ( projectId ) {
```

Comment:
Could you remove the spaces inside the`(`? So have `if (projectId) {` instead. Thanks